src/components/ShoppingList/ShoppingList.js

```diff
@@ -3,19 +3,14 @@ import { NavLink } from 'react-router-dom';
 
 import firebase from 'firebase/app';
 import { db } from '../../lib/firebase.js';
-import { useCollection } from 'react-firebase-hooks/firestore';
 
 import calculateEstimate from '../../lib/estimates.js';
 import { DateTime } from 'luxon';
 
 import ItemFilter from '../ItemFilter/ItemFilter.js';
 import ShoppingListItem from '../ShoppingListItem/ShoppingListItem.js';
 
-function ShoppingList({ listId, handleModalOpen }) {
-  const [listItems, loading, error] = useCollection(
-    db.collection(`lists/${listId}/items`),
-  );
```

Comment:
I moved this up to ListView.js as a parent so that I can pass listItems to both the ShoppingList.js component and the Header.js component

Reply:
Perfect! Good thinking ahead, as we need to have this in ListView for when we move the AddItemForm component in there as well!